pipeline-model-definition/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/actions/RestartDeclarativePipelineAction.java

```diff
@@ -26,14 +26,10 @@
 
 import hudson.Extension;
 import hudson.Util;
-import hudson.model.Action;
-import hudson.model.Cause;
-import hudson.model.CauseAction;
-import hudson.model.Failure;
-import hudson.model.Item;
+import hudson.model.*;
```

Comment:
Just so that we don't have IDE import updates running into each other, could you switch back from the wildcard imports, please?